php/Terminus/Commands/SiteCommand.php

```diff
@@ -1605,77 +1605,51 @@ public function setDrushVersion($args, $assoc_args) {
   }
 
   /**
-   * Add/replace an HTTPS Certificate for an environment
+   * Add/replace an HTTPS certificate for an environment
    *
    * ## OPTIONS
    *
    * [--site=<site>]
-   * : name of the site
+   * : Name of the site to apply the HTTPS certificate to
    *
    * [--env=<env>]
-   * : site environment
+   * : Environment of the site to apply the HTTPS certificate to
    *
    * [--certificate=<value>]
    * : Certificate
    *
    * [--private-key=<value>]
-   * : RSA Private Key
+   * : RSA private key
    *
    * [--intermediate-certificate=<value>]
-   * : (optional) CA Intermediate Certificate(s)
+   * : (optional) CA intermediate certificate(s)
    *
    * @subcommand set-https-certificate
    */
   public function setHttpsCertificate($args, $assoc_args) {
     $site        = $this->sites->get(
-      $this->input()->sitename(['args' => $assoc_args])
+      $this->input()->sitename(['args' => $assoc_args,])
     );
     $environment = $site->environments->get(
-      $this->input()->env(['args' => $assoc_args, 'site' => $site])
-    );
-
-    $certificate = $this->input()->string(
-      [
-        'args'     => $assoc_args,
-        'key'      => 'certificate',
-        'message'  => 'Certificate',
-      ]
-    );
-    $private_key = $this->input()->string(
-      [
-        'args'    => $assoc_args,
-        'key'     => 'private-key',
-        'message' => 'RSA Private Key',
-      ]
+      $this->input()->env(['args' => $assoc_args, 'site' => $site,])
     );
 
-    $is_interactive = !isset($assoc_args['certificate']);
-    if ($is_interactive) {
-      $intermediate_certificate = $this->input()->string(
-        [
-          'args'    => $assoc_args,
-          'key'     => 'intermediate-certificate',
-          'message' => 'CA Intermediate Certificate(s) (optional)',
-        ]
-      );
-    } else {
-      $intermediate_certificate = '';
-    }
-
-    $options = [
-      'certificate' => trim($certificate),
-      'private_key' => trim($private_key)
+    $https = [
+      'cert' => $this->input()->optional(
+        ['choices' => $assoc_args, 'key' => 'certificate',]
+      ),
+      'key' => $this->input()->optional(
+        ['choices' => $assoc_args, 'key' => 'private-key',]
+      ),
+      'intermediary' => $this->input()->optional(
+        ['choices' => $assoc_args, 'key' => 'intermediate-certificate',]
+      ),
```

Comment:
I just changed it so that it will ask users operating interactively for an intermediary certificate.